src/agent.ts

```diff
@@ -0,0 +1,54 @@
+import { Finding, HandleTransaction, TransactionEvent } from "forta-agent";
+import { ethers } from "ethers";
+import { createFinding } from "./findings";
+import {
+  NETHERMIND_DEPLOYER_ADDRESS,
+  AGENT_REGISTRY_ADDRESS,
+  CREATE_AGENT_ABI,
+  UPDATE_AGENT_ABI,
+  DISABLE_AGENT_ABI,
+} from "./constants";
+
+const provideHandleTransaction = (
+  deployerAddress: string,
+  agentRegistryAddress: string,
+  createAgentAbi: string,
+  updateAgentAbi: string,
+  disableAgentAbi: string
+): HandleTransaction => {
+  const iface = new ethers.utils.Interface([
+    createAgentAbi,
+    updateAgentAbi,
+    disableAgentAbi
+  ]);
+
+  return async (txEvent: TransactionEvent) => {
+    const findings: Finding[] = [];
+
+    if (txEvent.from !== deployerAddress.toLowerCase()) return findings;
+
+    // filter by relevant events
+    const agentDeploymentCalls = txEvent.filterFunction(
+      [createAgentAbi, updateAgentAbi, disableAgentAbi],
```

Comment:
If passing it as an array here, and they're not used individually elsewhere, you could declare an ABI array in `constants.ts` and use _that_ as an argument here.